### DISCUSSION TITLE
Question about speech conditioning

### DISCUSSION THREAD

**Discussion by asr-pub** [neonbjb/tortoise-tts#640] [General]
In the paper :
> Two encodings were produced for each training sample, which are averaged together.

Why use two encodings (one is self, the other is another clip of the same person speaking) ? How about use the single encoding of current sample ?

**Comment by manmay-nakhashi** [comment]
because if  we just use the same audio as conditioning it overfits into that data.

**Reply by asr-pub** [reply]
So, during each training iteration, for the same audio, its Speaker Condition varies, which means that SC is dynamically computed ?
